### DISCUSSION TITLE
06/10 안건

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#65] [Announcements]
안건이 있다면 작성해주세요😊

**Comment by NamgyungKim** [comment]
AI 기술배찌 추가

# AI
Python
java
C/C++
TensorFlow.js
R
Lua
Julia
Swift

**Reply by Hyevvy** [reply]
남경님 멋쪄요 ~~

**Comment by YuHyun-P** [comment]
✅ reset css -> 현진님이 해주시기로!

**Comment by YuHyun-P** [comment]
반응형 mixin 추가

**Comment by YuHyun-P** [comment]
폴더구조 컨벤션
 - styles.tsx, import * as S
 - assets 폴더 vs. 필요한 곳에 같이 위치

**Reply by Hyevvy** [reply]
<img width="295" alt="스크린샷 2022-06-10 오후 6 43 29" src="https://user-images.githubusercontent.com/72402747/173038642-b3bc61b7-e516-48cc-9648-0fff80cafcf7.png">

**Reply by Hyevvy** [reply]
<img width="285" alt="스크린샷 2022-06-10 오후 6 44 29" src="https://user-images.githubusercontent.com/72402747/173038756-be1da6b7-b832-47f0-b8d0-cc3a20ef8cc6.png">

**Reply by Hyevvy** [reply]
profile_default_image

**Reply by Hyevvy** [reply]
icon_cake_fill.svg

**Reply by YuHyun-P** [reply]
ProfileImage_default_image

**Reply by Hyevvy** [reply]
about_banner.png

**Reply by YuHyun-P** [reply]
profile_default_image

**Comment by Hyevvy** [comment]
[냐냥~~](https://github.com/prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung/discussions/72)

**Comment by NamgyungKim** [comment]
드롭다운 컴포넌트로 하나요?

**Reply by YuHyun-P** [reply]
공통 컴포넌트로~
